### PR TITLE
Fix broken unit tests in lms/djangoapps.

### PR DESCRIPTION
**Pull request by attiyaIshaque**

### CODE REVIEW

File: lms/djangoapps/instructor_task/tasks_helper/grades.py [hunk 1]
```diff
@@ -766,7 +766,7 @@ def generate(cls, _xmodule_instance_args, _entry_id, course_id, task_input, acti
         student_data, student_data_keys = cls._build_student_data(
             user_id=task_input.get('user_id'),
             course_key=course_id,
-            usage_key_str_list=problem_locations.split(','),
+            usage_key_str_list=[problem_locations.split(',')],
```
Comment (shafqatfarhan) [inline lms/djangoapps/instructor_task/tasks_helper/grades.py]: No need to use **[]** as string's split method splits a string into a list

File: lms/djangoapps/instructor_task/tests/test_tasks_helper.py [hunk 2]
```diff
@@ -538,12 +538,12 @@ def test_build_student_data_for_block_without_generate_report_data(self, mock_li
             student_data, _ = ProblemResponses._build_student_data(
                 user_id=self.instructor.id,
                 course_key=self.course.id,
-                usage_key_str=str(problem.location),
+                usage_key_str_list=[str(problem.location)],
```
Comment (shafqatfarhan) [inline lms/djangoapps/instructor_task/tests/test_tasks_helper.py]: `usage_key_str_list ` is not used in ironwood release (ref: https://github.com/edx/edx-platform/blob/441d6384eef5ed0b62e936a7d85958d10e43d92e/lms/djangoapps/instructor_task/tests/test_tasks_helper.py#L541)

File: lms/djangoapps/instructor_task/tests/test_tasks_helper.py [hunk 3]
```diff
@@ -598,7 +598,7 @@ def test_build_student_data_for_block_with_real_generate_report_data(self):
         student_data, _ = ProblemResponses._build_student_data(
             user_id=self.instructor.id,
             course_key=self.course.id,
-            usage_key_str=str(self.course.location),
+            usage_key_str_list=[str(self.course.location)],
```
Comment (shafqatfarhan) [inline lms/djangoapps/instructor_task/tests/test_tasks_helper.py]: `usage_key_str_list` is not used in ironwood release (ref: https://github.com/edx/edx-platform/blob/441d6384eef5ed0b62e936a7d85958d10e43d92e/lms/djangoapps/instructor_task/tests/test_tasks_helper.py#L541)

File: lms/djangoapps/instructor_task/tests/test_tasks_helper.py [hunk 4]
```diff
@@ -636,7 +636,7 @@ def test_build_student_data_for_block_with_generate_report_data_not_implemented(
 
     def test_success(self):
         task_input = {
-            'problem_location': str(self.course.location),
+            'problem_locations': str(self.course.location),
```
Comment (shafqatfarhan) [inline lms/djangoapps/instructor_task/tests/test_tasks_helper.py]: Please verify changes with upstream https://github.com/edx/edx-platform/blob/441d6384eef5ed0b62e936a7d85958d10e43d92e/lms/djangoapps/instructor_task/tests/test_tasks_helper.py#L639

**Comment by attiyaIshaque** [comment]
@shafqatfarhan I have addressed your comments and made changes according to upstream.

**Pull request merged**